src/components/layout/TitleHeader.js

```diff
@@ -1,14 +1,61 @@
 import React from 'react'
-import { Link } from 'react-router-dom'
+import { deurlify, makePascalCase } from '../../lib/helpers'
 
-export default function TitleHeader() {
+function getStaticTitleHeader() {
   return (
     <section className="hero is-info">
-      <div className="hero-body">
+      <div className="title-header hero-body">
         <div className="container has-text-centered">
-          <h1 className="title is-1">
-            <Link to={'/'}>Blockchain Workbench</Link>
-          </h1>
+          <h1 className="title is-1">Blockchain Workbench</h1>
+        </div>
+      </div>
+    </section>
+  )
+}
+
+export default function TitleHeader(props) {
+  if (!props.page || !props.category) {
+    return getStaticTitleHeader()
+  }
+  return (
+    <section className="hero is-info">
+      <div className="title-header hero-body">
+        <div className="container has-text-centered">
+          <p>
+            <a href={'http://'}>Courses</a> &#x203A; <a href="#">{deurlify(props.category)}</a>
+          </p>
+          <h1 className="title is-1">{props.page.title}</h1>
+        </div>
+        <div className={'level header-pageinfo'}>
+          <div className={'level-left'}>
+            <div>
+              <span className="icon">
+                <i className="fas fa-signal" />
+              </span>
+              <div>{makePascalCase(props.page.difficulty)}</div>
+            </div>
+            <div>
+              <span className="icon">
+                <i className="fas fa-clock" />
+              </span>
+              <div>{props.page.duration || 5} Min</div>
```

Comment:
this may not be in `min`. It could be hours. I think this should go into a dedicated function.

This could also help later if we want to infer duration from page content or collected data

Reply:
This also makes sense to get time for categories

Reply:
Can you mark it resolved once commited?

Reply:
extracted it into own function. we can improve that later, once we know how we want to handle the time or infer the duration